Tribler/Core/CacheDB/SqliteCacheDBHandler.py

```diff
@@ -492,6 +492,9 @@ def addOrGetTorrentIDSReturn(self, infohashes):
         for infohash, torrent_id in torrent_id_results.iteritems():
             if torrent_id is None:
                 to_be_inserted.add(infohash)
+        for infohash in infohashes:
```

Comment:
This is a hack, getTorrentIDS should return a list consisting of infohash, torrent_id for all infohashes in infohashes.
Fix getTorrentIDS instead.